public/sass/components/modal.scss

```diff
@@ -57,6 +58,74 @@ gu-box.modal__content {
   margin-left: 10px;
 }
 
+.modal__content__destination__list {
```

Comment:
The naming convention is BEM here, so each __ means the subcomponent. 

So if there's a `destination` component with a  `list` component within then that name is correct, but if there is a `destination list` component then it's clearer to use `.modal__content__destination-list`